scripts/qesap/test/e2e/test.sh

```diff
@@ -268,3 +273,14 @@ occurrence=$(grep -E "TASK \[Say hello\]" "${THIS_LOG}" | wc -l)
 [[ $occurrence -eq 1 ]] || test_die "Some Ansible stdout lines are repeated ${occurrence} times in place of exactly 1"
 set -e
 rm "${THIS_LOG}"
+
+test_step "[test_6.yaml] Run Ansible with playbooks"
+rm ansible.*.log.txt || echo "Nothing to delete"
+cp timbio.yaml "${QESAPROOT}/ansible/playbooks/timbio.yaml"
```

Comment:
timbio.yaml has not been added to the PR. 

But I think that none of the 3 new are needed. Please just do:
```
cp sambuconero.yaml "${QESAPROOT}/ansible/playbooks/timbio.yaml"
cp sambuconero.yaml "${QESAPROOT}/ansible/playbooks/buga.yaml"
cp sambuconero.yaml "${QESAPROOT}/ansible/playbooks/purace.yaml"
```

Reply:
Good point thx